smoker/server/plugins/__init__.py

```diff
@@ -738,6 +742,14 @@ def get_param(self, name, default=None):
         except KeyError:
             return default
 
+    def close_unnecessary_sockets(self):
+        """close unnecessary open sockets cloned on fork"""
+        open_sockets = filter(lambda x: type(x) == socket._socketobject, gc.get_objects())
+        for cur_socket in open_sockets:
+            # close all sockets except udp (type 2) - udp is used by Syslog
+            if cur_socket.type != 2:
```

Comment:
I'd rather see the logic inverted, close all TCP and leave everything else.
Also, no need to use the magic constant of two, the `socket` module offeres the constants predefined https://docs.python.org/2/library/socket.html#socket.SOCK_STREAM